Classes/Driver/IndexDriverInterface.php

```diff
@@ -19,13 +19,22 @@
  */
 interface IndexDriverInterface
 {
+
     /**
      * Get the list of Indexes attached to the given alias
      *
      * @param string $alias
      * @return array
      */
-    public function indexesByAlias(string $alias): array;
+    public function getIndexeNamesByAlias(string $alias): array;
+
```

Comment:
I guess it should be `getIndexNamesByAlias` (same for `getIndexNamesByPrefix`? Sorry for the nitpick, but as it's in the Interface...

Reply:
uuh, fixed, thanks.